<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CardGenerator</string>
  </property>

  <widget class="QWidget" name="centralwidget">

   <layout class="QHBoxLayout" name="mainLayout">

    <!-- LEFT PANEL -->
    <item>
     <widget class="QFrame" name="leftPanel">
      <property name="minimumWidth">
       <number>260</number>
      </property>
      <property name="frameShape">
       <enum>QFrame.StyledPanel</enum>
      </property>

      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QLabel" name="lblCards">
         <property name="text">
          <string>Список карт</string>
         </property>
        </widget>
       </item>

       <item>
        <widget class="QListWidget" name="listCards"/>
       </item>
      </layout>
     </widget>
    </item>

    <!-- CENTER CANVAS -->
    <item>
     <widget class="QFrame" name="canvasFrame">
      <property name="frameShape">
       <enum>QFrame.Box</enum>
      </property>
      <layout class="QVBoxLayout" name="canvasLayout">
       <item>
        <widget class="QLabel" name="lblCanvas">
         <property name="text">
          <string>Canvas (Drag &amp; Drop)</string>
         </property>
         <property name="alignment">
          <set>Qt.AlignHCenter</set>
         </property>
        </widget>
       </item>

       <item>
        <widget class="QWidget" name="canvasWidget">
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>800</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">
           background-color: #1e1e1e;
           border: 1px solid #555;
          </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <!-- RIGHT PANEL -->
    <item>
     <widget class="QFrame" name="rightPanel">
      <property name="minimumWidth">
       <number>260</number>
      </property>
      <property name="frameShape">
       <enum>QFrame.StyledPanel</enum>
      </property>

      <layout class="QVBoxLayout" name="rightLayout">

       <item>
        <widget class="QLabel" name="lblProperties">
         <property name="text">
          <string>Параметри</string>
         </property>
        </widget>
       </item>

       <item>
        <widget class="QTextEdit" name="propertyEditor">
         <property name="placeholderText">
          <string>Тут буде панель властивостей…</string>
         </property>
        </widget>
       </item>

      </layout>
     </widget>
    </item>

   </layout>
  </widget>

  <!-- BOTTOM BUTTONS -->
  <widget class="QWidget" name="bottomWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>830</y>
     <width>1400</width>
     <height>70</height>
    </rect>
   </property>

   <layout class="QHBoxLayout" name="bottomLayout">
    <item>
     <widget class="QPushButton" name="btnLoadJSON">
      <property name="text">
       <string>Load JSON</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QPushButton" name="btnSetWorkspace">
      <property name="text">
       <string>Set Workspace</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QPushButton" name="btnGeneratePreview">
      <property name="text">
       <string>Generate Preview</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QPushButton" name="btnGenerateSet">
      <property name="text">
       <string>Generate Set</string>
      </property>
     </widget>
    </item>

    <item>
     <widget class="QPushButton" name="btnGeneratePDF">
      <property name="text">
       <string>Generate PDF</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>

 </widget>

 <resources/>
 <connections/>
</ui>
